test/e2e/client_test.go

```diff
@@ -106,3 +115,58 @@ func (t *testClient) cleanupProject(timeout time.Duration) error {
 	}
 	return nil
 }
+
+func (t *testClient) dumpInfo() error {
+	// gather events
+	eventList, err := t.kc.CoreV1().Events(t.namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	eventBuf := bytes.NewBuffer(nil)
```

Comment:
whats the difference/reason in doing it this way vs doing it like `var eventBuf bytes.Buffer` or `eventBuf := new(bytes.Buffer)`?

Reply:
There isn't any difference.